reference/kubectl-plugin.md

```diff
@@ -2,15 +2,6 @@
 
 The ‘Mayastor kubectl plugin’ can be used to view and manage Mayastor resources such as nodes, pools and volumes. It is also used for operations such as scaling the replica count of volumes. 
 
-### Prerequisites
```

Comment:
Are we removing this prerequisite for 2.0.0?

Reply:
Yes, looks like it. Got this input from Abhinandan.